src/lib/components/Footer/Footer.svelte

```diff
@@ -92,21 +90,14 @@
   <div class="usa-footer__secondary-section ldaf-footer__secondary-section">
     <div class="grid-container">
       <div class="grid-row grid-gap">
-        <div
-          class="usa-footer__logo ldaf-footer__logo grid-row tablet:grid-col-6 desktop:grid-col-8 grid-gap-2"
-        >
-          <div class="tablet:grid-col-auto">
-            <div class="usa-footer__logo-img">
-              <Logo />
-            </div>
+        <div class="usa-footer__logo grid-row tablet:grid-col-6 desktop:grid-col-8 grid-gap-2">
+          <!-- Vertical, stacked logo for mobile. -->
+          <div class="display-block tablet:display-none">
+            <Logo placement="mobile-footer" />
```

Comment:
Opted not to provide extra content here, as it's not a link and only serves an aesthetic purpose. Screen readers will still inform users that there's a logo here.